lib/gds_api/email_alert_api.rb

```diff
@@ -97,6 +97,15 @@ def unsubscribe(uuid)
     post_json("#{endpoint}/unsubscribe/#{uuid}")
   end
 
+  # Subscribe
+  # #
+  # @param attributes [Hash] address, subscribable_id
+  #
+  # @return [Hash] subscription_id
+  def subscribe(attributes)
```

Comment:
I'd be tempted to make this `def subscribe(subscribable_id:, address:)` so we get some documentation and we can leave it Ruby to handle the argument validation for us.

Reply:
I was making it consistent with the other methods but I agree... Fixed in 1f67417